src/datajudge/db_access.py

```diff
@@ -777,6 +777,39 @@ def column_operator(column):
     return get_column(engine, ref, aggregate_operator=column_operator)
 
 
+def get_kth_percentile(engine, ref, k):
+    row_count = "row_count"
+    row_num = "row_num"
```

Comment:
```suggestion
    ROW_COUNT = "dj_row_count"
    ROW_NUM = "dj_row_num"
```
Two changes
1. Capitalized, to show the intention that these strings are a constant.
2. Use the prefix `dj` to make it explicit that these are arbitrary ones from datajudge.

I find the first more relevant than the second. I'm open to keeping them as they're given a good argument.

Reply:
I'm fully with you on the second point!

Regarding the first point, PEP8 [0] says:

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

I would've read this as if it would be more appropriate to lower-case the aforementioned objects. Do you disagree?

[0] https://peps.python.org/pep-0008/#constants